docs/principes.md

```diff
@@ -0,0 +1,67 @@
+# Some principes
+Here are some principes we're trying to use when developing moon. Why do we think it's important? Well, in most cases bad principes lead to a bad architecture. Bad architecture leads to a buggy, unusable, unstable and ugly code. So, to be aware of that, let's discuss some points that seems to be important for me.
+
+
+## Our main goals are:
+- create multi-technology design system
+- ... to make frontend development easier for our consumers
+- ... and make library development transparent, involve library consumers in feature discussion and/or implementing
+
+## Programming rules:
+We do preffer and respect the following principes:
+Headless - as soon as we're using few techologies, we need to use headless approach, so sub-components structure & interactions are mostly predefined when creatig components for each technology.
```

Comment:
"we're using few techologies" what does this mean?

Reply:
sorry, some kind of draft documentation is here right now. 
please let me expand those thoughts before we'll start a discussion.

few technologies - meant React, Surface & Flutter